Brokerages/Zerodha/Kite.cs

```diff
@@ -0,0 +1,1081 @@
+                Utils.AddIfNotNull(param, "disclosed_quantity", DisclosedQuantity.ToString());
+            }
+            else if (VarietyString != "co" && ProductString != "co")
+            {
+                Utils.AddIfNotNull(param, "exchange", Exchange);
+                Utils.AddIfNotNull(param, "tradingsymbol", TradingSymbol);
+                Utils.AddIfNotNull(param, "transaction_type", TransactionType);
+                Utils.AddIfNotNull(param, "quantity", Quantity.ToStringInvariant());
+                Utils.AddIfNotNull(param, "price", Price.ToString());
+                Utils.AddIfNotNull(param, "product", Product);
+                Utils.AddIfNotNull(param, "order_type", OrderType);
+                Utils.AddIfNotNull(param, "validity", Validity);
+                Utils.AddIfNotNull(param, "disclosed_quantity", DisclosedQuantity.ToString());
+            }
+
+            return Put("orders.modify", param);
+        }
+
+        /// <summary>
+        /// Cancel an order
+        /// </summary>
+        /// <param name="OrderId">Id of the order to be cancelled</param>
+        /// <param name="Variety">You can place orders of varieties; regular orders, after market orders, cover orders etc. </param>
+        /// <param name="ParentOrderId">Id of the parent order (obtained from the /orders call) as BO is a multi-legged order</param>
+        /// <returns>Json response in the form of nested string dictionary.</returns>
+        public JObject CancelOrder(string OrderId, string Variety = Constants.VARIETY_REGULAR, string ParentOrderId = null)
+        {
+            var param = new Dictionary<string, dynamic>();
+
+            Utils.AddIfNotNull(param, "order_id", OrderId);
+            Utils.AddIfNotNull(param, "parent_order_id", ParentOrderId);
+            Utils.AddIfNotNull(param, "variety", Variety);
+
+            return Delete("orders.cancel", param);
+        }
+
+        /// <summary>
+        /// Gets the collection of orders from the orderbook.
+        /// </summary>
+        /// <returns>List of orders.</returns>
+        public List<Order> GetOrders()
+        {
+            var ordersData = Get("orders");
+
+            List<Order> orders = new List<Order>();
+
+            foreach (JObject item in ordersData["data"])
+                orders.Add(new Order(item));
+
+            return orders;
+        }
+
+        /// <summary>
+        /// Gets information about given OrderId.
+        /// </summary>
+        /// <param name="OrderId">Unique order id</param>
+        /// <returns>List of order objects.</returns>
+        public List<Order> GetOrderHistory(string OrderId)
+        {
+            var param = new Dictionary<string, dynamic>
+            {
+                { "order_id", OrderId }
+            };
+
+            var orderData = Get("orders.history", param);
+
+            List<Order> orderhistory = new List<Order>();
+
+            foreach (JObject item in orderData["data"])
```

Comment:
- Please always uses braces;
 - Please ensure every method is commented (exceptions above are not)
 - Please align all /// comments
 - Camel case variables; orderhistory
 - Follow our MS code naming conventions - var when possible.
 - Use LEAN SHA256 and UnixTime  utils methods (to avoid duplication)
We might be able to use this in live trading -- pretty exciting.

Reply:
Updated the pull request with the changes.